src/Drupal/ModuleTrait.php

```
@@ -0,0 +1,299 @@
+   *   | views  |
+   * @endcode
+   *
+   * @Given the following modules are enabled:
+   */
+  public function moduleEnsureEnabledMultiple(TableNode $modules_table): void {
+    foreach ($modules_table->getColumn(0) as $module) {
+      if (!$this->moduleIsEnabled($module)) {
+        $this->moduleEnable($module);
+      }
+    }
+  }
```

PHPMD (2.15.0)

The parameter $modules_table is not named in camelCase. (undefined)

(CamelCaseParameterName)